src/answers-headless.ts

```diff
@@ -81,6 +81,12 @@ export default class AnswersHeadless {
    * @param verticalKey - The vertical key to set
    */
   setVertical(verticalKey: string): void {
+    this.setState({
+      ...this.state,
+      filters: {},
```

Comment:
Are there other things in state that we might want to clear? For example, it might make sense to clear DirectAnswer as well

Reply:
Yeah, I updated to also clear direct answer, query rules, and search status